source/localizable/routing/defining-your-routes.md

```diff
@@ -84,7 +84,18 @@ the `{{outlet}}` of the `posts` template.
 A nested route's names includes the names of its ancestors.
 If you want to transition to a route (either
 via `transitionTo` or `{{#link-to}}`), make sure to use the full route
-name (`posts.new`, not `new`).
+name (`posts.new`, not `new`). If you don't want to use the full route
+name (e.g., because a resource can be accessed in multiple locations), use the
```

Comment:
This is exactly why this section was removed. You CANNOT use the same route name for two routes. The only thing that restNamespace gives you is a shorter name.